subsys/net/lib/websocket/Kconfig

```diff
@@ -9,6 +9,7 @@ menuconfig WEBSOCKET_CLIENT
 	select HTTP_CLIENT
 	select MBEDTLS
 	select BASE64
+	select PSA_WANT_ALG_SHA_1
```

Comment:
There's no such config in Zehpyr (it's NCS only).

Reply:
I see thanks for the feedback, in Zephyr there is MBEDTLS_MAC_SHA1_ENABLED, but that does not work when the TFM of NCS is enabled. Do you know a universal way that works both with and without NCS?

Reply:
Currently, sadly, there's no common way to handle mbed TLS configuration. For zephyr, and mbed TLS shipped with Zephyr, you could use `select MBEDTLS_MAC_SHA1_ENABLED if MBEDTLS_BUILTIN`